rfcs/onboard_infrastructure_providers.md

```diff
@@ -0,0 +1,48 @@
+### Introduction
+
+It seems reasonable that being Paseo a community testnet not a single party has to make all the efforts of maintaining this platform. At the same time, moving away from a single operator should not compromise the availablity of the network.
+
+This RFC talks about how different infrastructure providers can support Paseo network.
+
+### Background
+
+Different members of the Polkadot ecosystem have expressed interest to participate in this initiative. A selection criteria is necessary to don't compromise the network while allowing as many parties as possible. 
+
+Infrastructure providers can run validators following the information detailed in [#20](https://github.com/paseo-network/paseo-rfcs/issues/20).
+
+### Proposal
+
+Interested infrastructure providers (referred as _Contributor_ or _the contributor_ from here on) should open a PR in this repository and label is as `infra onboarding`.
+
+In this PR they should create a file as described in [#20](https://github.com/paseo-network/paseo-rfcs/issues/20):
+
+```bash
+paseo-rfcs
+└── paseo-validators
+     ├── existing-infra-prov.yml
+     └── <your-team-name>.yml
+```
+This file should, at least, contain the identity of the contributor.
+
+The **election criteria** is:
+
+- Contributor is part of [Polkadot 1KV Programme](https://wiki.polkadot.network/docs/thousand-validators).
```

Comment:
- Contributor is part of the [Polkadot 1KV Programme](https://wiki.polkadot.network/docs/thousand-validators).